pkg/job/local_job.go

```diff
@@ -70,14 +70,24 @@ func (j *localJob) AbortInstallConfig(ctx context.Context, cluster common.Cluste
 
 func (j *localJob) GenerateISO(ctx context.Context, cluster common.Cluster, jobName string, imageName string, ignitionConfig string, eventsHandler events.Handler) error {
 	log := logutil.FromContext(ctx, j.log)
-	envVars := append(os.Environ(),
+	workDir := "/data"
+	cmd := exec.Command(workDir + "/assisted-iso-create")
+	cmd.Env = append(os.Environ(),
 		"S3_ENDPOINT_URL="+j.Config.S3EndpointURL,
 		"IGNITION_CONFIG="+ignitionConfig,
 		"IMAGE_NAME="+imageName,
+		"COREOS_IMAGE="+workDir+"/livecd.iso",
 		"S3_BUCKET="+j.Config.S3Bucket,
-		"aws_access_key_id="+j.Config.AwsAccessKeyID,
-		"aws_secret_access_key="+j.Config.AwsSecretAccessKey,
-		"WORK_DIR=/data",
+		"AWS_ACCESS_KEY_ID="+j.Config.AwsAccessKeyID,
```

Comment:
there is a new env variable that needs to be passed to the executable: S3_REGION
please see pkg/job/job.go, line 240

Reply:
Ok, that's going to go away when we switch to local storage soon for local jobs anyway.